app/src/main/java/com/ccc/nameapp/extension/ActivityExtension.kt

```diff
@@ -107,7 +107,8 @@ fun Activity.isKeyboardOpened(): Boolean {
 }
 
 fun Activity.observerKeyboardShow(onKeyboardShow: (Boolean, ViewTreeObserver.OnGlobalLayoutListener) -> Unit) {
-    getRootView().viewTreeObserver.addOnGlobalLayoutListener(object : ViewTreeObserver.OnGlobalLayoutListener {
+    getRootView().viewTreeObserver.addOnGlobalLayoutListener(object :
+        ViewTreeObserver.OnGlobalLayoutListener {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/9cbff12fc6911fd41007439107765dbec2127af9/app/src/main/java/com/ccc/nameapp/extension/ActivityExtension.kt#L111"></span>Unexpected indentation (8) (should be 12)</td>
    </tr>
  </tbody>
</table>